hail/scripts/deploy.sh

```diff
@@ -8,15 +8,16 @@ retry() {
         (sleep 5 && "$@");
```

Comment:
update hail/Makefile as well with the extra argument. That's used in an emergency situation when we need to hand deploy.

Reply:
Done!